core/src/backend/navigator.rs

```diff
@@ -682,3 +690,14 @@ pub fn fetch_path<NavigatorType: NavigatorBackend>(
         Ok(response)
     })
 }
+
+/// Parses and returns the encoding out of an HTTP header content type string
+/// if existing.
+pub fn get_encoding(content_type: String) -> Option<&'static Encoding> {
+    let content_vector = content_type.split("charset=").collect::<Vec<&str>>();
```

Comment:
I think you can call `split_once` and skip a Vec this way?

Reply:
You're right; thanks! :)